ads/google/doubleclick.js

```diff
@@ -164,6 +172,12 @@ function doubleClickWithGlade(global, data, gladeExperiment) {
   if (gladeExperiment === GladeExperiment.GLADE_EXPERIMENT) {
     jsonParameters.gladeExp = '1';
   }
+  const expIds = data['experimentId'];
+  if (expIds) {
+    jsonParameters.gladeEids = jsonParameters.gladeEids ?
+        jsonParameters.gladeEids + ',' + expIds : expIds;
```

Comment:
Oh, quickly: Is there a submission dependency here?  Does @mlb7687 's PR _have_ to merge before this one?  Or is it okay if they end up in different cuts?